python/ray/serve/api.py

```diff
@@ -1800,7 +1800,16 @@ def run(
     ]
 
     client.deploy_group(parameter_group, _blocking=True)
-    return deployments[-1].get_handle()
+
+    serve_handle = []
```

Comment:
confusing naming, how about:
`exposed_deployments = []`

then do `return exposed_deployments[0].get_handle()` at the end if assertion passes?

Reply:
sg